pkg/core/resources/apis/mesh/dataplane_validator_test.go

```diff
@@ -281,6 +360,91 @@ var _ = Describe("Dataplane", func() {
 				},
 			},
 		}),
+		Entry("legacy - invalid inbound interface", testCase{
+			dataplane: func() core_mesh.DataplaneResource {
```

Comment:
To me, this approach to defining test data is too cryptic.

I find it much easier to understand and maintain a complete snippet of `Dataplane` `YAML` than mentally merge these changes

Reply:
refactored